tests/validation/tests/v3_api/test_monitoring.py

```diff
@@ -86,7 +87,9 @@
     "workload": workload_graph_list,
     "node": node_graph_list,
 }
+STORAGE_CLASS = "longhorn"
```

Comment:
Not too familiar with storage/volumes but shouldn't we also make this `STORAGE_CLASS` available through environment variables?

Reply:
At this point we are testing only storage class set to longhorn.

Reply:
Sounds good then 👍